src/nav/components.yml

```diff
@@ -4,7 +4,13 @@
       title: Overview
 - title: Components
   items:
+    - title: Avatar
+      id: /components/avatar
+    - title: Breadcrumbs
+      id: /components/breadcrumbs
     - title: Buttons
       items:
-        - id: /components/button
+        - id: /components/buttons/anchor
+          title: Anchor
+        - id: /components/buttons/button
```

Comment:
Even though these segments are nested in the sidebar, do we want them to be under a `/components/buttons/*` url? It may make sense to have everything directly under `/components/*` if possible.

Reply:
We don't want the URL structure to reflect the IA?

Reply:
The sidebar is just a visual representation, we can have any IA that fits. With `components/buttons/*` that could imply that there is a root `components/buttons` page which includes an overview. Would we have `components/ranges/range` and `components/ranges/multi-thumb-range`? I'm not sure this scales to all packages.

i.e. `components/accordions/stepper`, `components/breadcrumbs`

Reply:
Great, great callout. Will amend.